src/darwin/Framework/chip_xcode_build_connector.sh

```diff
@@ -103,6 +103,7 @@ declare -a args=(
     'chip_enable_python_modules=false'
     'chip_device_config_enable_dynamic_mrp_config=true'
     'chip_log_message_max_size=4096' # might as well allow nice long log messages
+    'chip_logging_backend="none"'    # os_log() is integrated via CHIP_SYSTEM_CONFIG_PLATFORM_LOG
```

Comment:
So with this change, what happens with framework logs in CI?  For example when running the darwin unit tests.

Similarly, what happens with logs while running darwin-framework-tool?  Will logs from the tool itself go to stdout but not logs from the framework it links against?

Reply:
Hmm I'll have to look into that... but my initial answer would be "whatever happens now" because the current LoggingImpl for Darwin was also a no-op.

Reply:
darwin-framework-tool installs log redirect callbacks into the framework and into the copy of CHIP logging that's included directly, so that will continue to work as before.

Framework logs in unit tests go into os_log and get picked up from there by xctest as before.